### ISSUE TITLE
MacOS inputs lost on startup

### ISSUE THREAD

**Issue by palmerj** [mamedev/mame#10612]
When starting mame on MacOS in fullscreen mode and you move your mouse on startup, the application hangs and never gets to the OSD, and the mouse and normal keyboard input do not work If you don't move your mouse, it will start correctly! If the application hangs and you press CMD+H and minimise the app and then select it again to gain focus on the app, the OSD will then load. I've tested all video driver options (opengl, bgfx, metal), and other options with the mouse but I need help to resolve this issue. 

Note in playing around and changing m_mouse_captured to true in https://github.com/mamedev/mame/blob/master/src/osd/sdl/window.cpp#L1168 will at least get the OSD to load. However, inputs are still lost on the startup OSD. So if you are loading a rom and skip the info OSD then you can play the rom ok with inputs being received.  

Notes:

* I'm running MacOS 12.6
* I'm running SDL2-2.26.0
* I've complied from source using SDL2 framework and also installed from homebrew, but the results are the same.

**Comment by rb6502** [comment]
That's likely a bug in SDL itself.  I would very much like to get OSD=mac builds to work where MAME would be Cocoa-native, but I'm pretty noob at Cocoa so it hasn't gone great so far.

**Comment by rb6502** [comment]
Right, video doesn't work for OSD=mac, which is why it's not normally enabled :-)

Debugging's usually done with command line LLDB, as our build system currently can't output a working Xcode project.  We're looking to switch to CMake, which would make that possible, but there's no timeline for that yet.

**Comment by palmerj** [comment]
Fixed
